services/kommander/0.2.0/defaults/cm.yaml

```diff
@@ -56,12 +56,12 @@ data:
           manager:
             rootCertSecretName: ${caSecretName}
             image:
-              tag: ${licensingControllerManagerImageTag}
-              repository: ${licensingControllerManagerImageRepository}
+              tag: ${kommanderLicensingControllerManagerImageTag:=v2.2.0-dev}
+              repository: ${kommanderLicensingControllerManagerImageRepository}
       webhook:
         image:
-          tag: ${licensingControllerWebhookImageTag}
-          repository: ${licensingControllerWebhookImageRepository}
+          tag: ${kommanderLicensingControllerWebhookImageTag:=v2.2.0-dev}
```

Comment:
This is fixed in https://github.com/mesosphere/kommander/pull/1512.

Reply:
See my comment above.

Reply:
I've removed this now.